_includes/templates/compatibility-page.md

```diff
@@ -10,173 +12,183 @@ block chain space used, this is the most efficient form of fee bumping.
+    Does not visually indicate that a received transaction has signaled RBF when viewing the transaction details
   {% when "na" %}{:.feature-neutral}
-  - **FIXME**<br>
-    FIXME
+  - **Does not show transaction details**<br>
+    This service does not show transaction details natively. Usually this means the service links off to a block explorer for transaction details.
   {% when "untested" %}{:.feature-neutral}
-  - **FIXME**<br>
-    We didn’t test this
+  - **Not tested**<br>
+    We either didn’t test this or could not appropriately determine the results
   {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
 {% endcase %}
 
-{% case rbf.receive.shows_original_version %}
-  {% when "true" %}{:.feature-good}
-  - **Shows replaced transactions**<br>
-    FIXME
-  {% when "false" %}{:.feature-bad}
-  - **Doesn't show replaced versions**<br>
-    FIXME
-  {% when "na" %}{:.feature-neutral}
-  - **FIXME**<br>
-    FIXME
-  {% when "untested" %}{:.feature-neutral}
-  - **FIXME**<br>
-    We didn’t test this
-  {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
-{% endcase %}
+{:id="receive-replaced"}
+{% if rbf.receive.shows_replaced_version == "true" and rbf.receive.shows_original_version == "true"  %}
+  {:.feature-good}
+  - **Shows replacement and original transactions**<br>
+    Both the original transaction and replacement transactions are shown in the
+    transaction list
+{% elsif rbf.receive.shows_replaced_version == "na" or
+    rbf.receive.shows_original_version == "na" %}
+  {:.feature-neutral}
+  - **No transaction list**<br>
+    This service does not support listing of transactions
+{% elsif rbf.receive.shows_replaced_version == "untested" or
+    rbf.receive.shows_original_version == "untested" %}
+  {:.feature-neutral}
+  - **Not tested**<br>
+    We either didn’t test this or could not appropriately determine the results
+{% elsif rbf.receive.shows_replaced_version == "true" %}
+  {:.feature-good}
+  - **Shows replacement transaction only**<br>
+    Only the replacement transaction is shown in the transaction list. No original
+    transaction shown
+{% elsif rbf.receive.shows_original_version == "true" %}
+  {:.feature-bad}
+  - **Shows original transaction only**<br>
+    Only the original transaction shown in transaction list. Repacement transactions
+    not shown
+{% elsif rbf.receive.shows_original_version == "false" and rbf.receive.shows_replaced_version == "false" %}
+  {:.feature-bad}
+  - **No unconfirmed transactions**<br>
+    Neither the original nor replacement transactions are shown in the
+    transaction list. Most likely unconfirmed transactions are not supported
+{% else %} {% include ERROR_42_UNEXPECTED_VALUE %}
+{% endif %}
 
 ### Sending support
 
+{:id="send-signals_bip125"}
 {% case rbf.send.signals_bip125 %}
   {% when "true" %}{:.feature-good}
-  - **Signals BIP125 by default**<br>
-    FIXME
+  - **Allows sending RBF transactions**<br>
+    Allows sending of RBF transactions in the interface
```

Comment:
I think this description should explicitly mention BIP125 signaling eg "Allows sending of transactions which signal that they are BIP 125 replaceable" or similar